### ISSUE TITLE
The plugin doesn't generate all the fabrication output files with KiCad 6.0.1-0

### ISSUE THREAD

**Issue by Razerban** [Bouni/kicad-jlcpcb-tools#105]
**Describe the bug**
After upgrading to KiCad 6.0.1-0, the plugin doesn't generate all the fabrication output files.
The plugin was working with KiCad 6.0.0.

**Error**
```
Traceback (most recent call last):
  File "/Users/XXXX/Documents/KiCad/6.0/3rdparty/plugins/kicad-jlcpcb-tools/mainwindow.py", line 649, in generate_fabrication_data
    self.fabrication.generate_excellon()
  File "/Users/XXXX/Documents/KiCad/6.0/3rdparty/plugins/kicad-jlcpcb-tools/fabrication.py", line 197, in generate_excellon
    drlwriter.SetOptions(mirror, minimalHeader, offset, mergeNPTH)
  File "/Applications/KiCad/KiCad.app/Contents/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pcbnew.py", line 7016, in SetOptions
    return _pcbnew.EXCELLON_WRITER_SetOptions(self, aMirror, aMinimalHeader, aOffset, aMerge_PTH_NPTH)
TypeError: in method 'EXCELLON_WRITER_SetOptions', argument 4 of type 'wxPoint const &'
```

# KiCad Version

```
Application: KiCad

Version: (6.0.1-0), release build

Libraries:
	wxWidgets 3.1.5
	libcurl/7.77.0 SecureTransport (LibreSSL/2.8.3) zlib/1.2.11 nghttp2/1.42.0

Platform: macOS Version 12.1 (Build 21C52), 64 bit, Little endian, wxMac

Build Info:
	Date: Jan 14 2022 14:47:36
	wxWidgets: 3.1.5 (wchar_t,wx containers)
	Boost: 1.76.0
	OCC: 7.6.0
	Curl: 7.64.1
	ngspice: 35
	Compiler: Clang 12.0.0 with C++ ABI 1002

Build settings:
	KICAD_USE_OCC=ON
	KICAD_SPICE=ON

```

**Comment by Bouni** [comment]
I see what's the problem here.
I check for version 6.0.0 and didn't take into account that there will be bug fix releases with versions like 6.0.1 and so on.

I'm going to change this and check for 6.99 instead because that'll not change.

**Comment by Bouni** [comment]
https://github.com/Bouni/kicad-jlcpcb-tools/blob/main/fabrication.py#L192-L195

**Comment by Bouni** [comment]
Fixed in release 2022.01.01

**Comment by Razerban** [comment]
> Fixed in release 2022.01.01

Thanks, that indeed fixed the issue.
